### DISCUSSION TITLE
indexing files alphabetically

### DISCUSSION THREAD

**Discussion by KatePimbleton** [pschatzmann/arduino-audio-tools#906] [Q&A] [answered]
Hi,

I'm using "  player.setIndex(); " in order to play a .wav file.
is there a way to order the index alphabetically ?

Cheers

**Comment by pschatzmann** [comment, marked as answer]
You can use the indexed version and sort the index file
Or you can use your own custom AudioSource or the AudioSourceCallback to return the files sorted...
